File: backend/components/metiers/threed/tests_centralises/legacy/samples/monitoring/test_monitoring_sample.py
"""
Test ultra avancé pour monitoring_sample.py
"""

from threed.legacy.samples.monitoring.monitoring_sample import (
    monitoring_sample,
)


def test_monitoring_alert():
    assert "ALERT" in monitoring_sample("cpu", 150)


def test_monitoring_ok():
    assert "OK" in monitoring_sample("cpu", 50)


# ...autres tests avancés, edge cases, robustesse...

File: backend/components/metiers/threed/tests_centralises/legacy/samples/monitoring/test_monitoring_sample.js
// Test ultra avancé pour monitoring_sample.js
const monitoringSample = require('../../../legacy/samples/monitoring/monitoring_sample');

describe('Monitoring Sample', () => {
  it('doit alerter si la valeur dépasse 100', () => {
    expect(monitoringSample('cpu', 150)).toMatch(/ALERT/);
  });
  it('doit indiquer OK si la valeur est normale', () => {
    expect(monitoringSample('cpu', 50)).toMatch(/OK/);
  });
  // ...autres tests avancés, edge cases, robustesse...
});
